kahuna/public/js/search/results.js

```diff
@@ -370,6 +370,23 @@ results.controller('SearchResultsCtrl', [
             }
         });
 
+        const freeImageDeleteListener = $rootScope.$on('image-deleted', (e, image) => {
```

Comment:
Bit crappy to have to do this kind of stuff here, but agree it's necessary until we tidy up all this stateful business.